functions/lib/emails/email_reminder.py

```diff
@@ -0,0 +1,139 @@
+import json
+import time
+import uuid
+from dataclasses import dataclass
+from datetime import date, datetime, timedelta
+
+import pytz
+import requests
+from firebase_admin import firestore
+from firebase_functions import https_fn, options, scheduler_fn
+from google.cloud import firestore
+from google.cloud.firestore import DocumentReference, Transaction
+from google.protobuf.timestamp_pb2 import Timestamp
+from lib.auth import *
+from lib.constants import *
+from lib.emails.constants import LOOPS_API_KEY
+from lib.logging import Logger
+
+ACTIVE_PUBLIC = "Events/Active/Public"
```

Comment:
The same constant also exists in move_inactive_events. Could we merge all these constants into a central location.